### PR TITLE
reformatting, new module docstring format, some pylint fixes

### PR DESCRIPTION
**Pull request by Matuiss2**
This PR is for the basic_models branch

**Reformatting**

- Used black module for it

 **new module docstring format**
- Use this format
```
 """
@file: module file name
Created on created date
@project: project name
@author:  the module contributors

Module description
"""
```

**pylint fixes**

- Import order should be standard modules -> third-party modules -> local modules

- Unused variables that can't be removed(for example in tuple unpacking) should be defined as `_`

### CODE REVIEW

File: leelastar/network/layers.py [hunk 1]
```diff
@@ -48,20 +55,10 @@ def __init__(self, channels, kernel_size, se_ratio):
         super().__init__(
             OrderedDict(
                 [
-                    (
-                        "conv_layer_1",
-                        nn.Conv2d(
-                            channels, channels, kernel_size, padding=1, bias=False
-                        ),
-                    ),
+                    ("conv_layer_1", nn.Conv2d(channels, channels, kernel_size, padding=1, bias=False)),
```
Comment (kiudee) [inline leelastar/network/layers.py]: My black breaks this line (default configuration), do you use a longer line width?

Comment (Matuiss2) [reply leelastar/network/layers.py]: the line is on 120 length

Comment (Matuiss2) [reply leelastar/network/layers.py]: -l 120

Comment (kiudee) [reply leelastar/network/layers.py]: Ok, will make the change in my configuration

**Pull request merged**